docs/new/posts/MySQL/percona-xtradb-cluster-8.0.42-33-release.md

```diff
@@ -31,7 +31,7 @@ Improves State Snapshot Transfer (SST) failure diagnostics. `garbd` now uses dis
 
 * Improves optimizer behavior by restoring correct handling of const tables in `test_quick_select()`. A MySQL Upstream refactor (commit [9a13c1c](https://github.com/percona/percona-server/commit/9a13c1c6971f4bd56d143179ecfb34cca8ecc018){:target="_blank"}) removed the `QEP_TAB` dependency, causing `get_quick_record_count()` to no longer pass const table information. This could lead to suboptimal range scan boundaries. The applied patch resolves the issue by explicitly passing `const_tables` to `test_quick_select()`, ensuring consistent behavior with the pre-refactor logic.
 
-* Adds changes to the list of [MyRocks variables](https://docs.percona.com/percona-server/8.0/variables.html){:target="_blank"}. 
+* Adds changes to the list of [MyRocks variables](https://docs.percona.com/percona-server/8.0/variables.html){:target="_blank"}.
```

Comment:
For links, it's better to make them longer for users; longer links can provide more context about the destination. Search engines often favor descriptive links. Descriptive links can enhance accessibility for users who rely on screen readers. Longer links can reduce confusion by clearly indicating the purpose of the link. 
and so on.

Reply:
fixed